src/types/plugin-input.ts

```diff
@@ -33,6 +33,10 @@ export const pluginSettingsSchema = T.Object(
          * If set to false, the plugin runs even if the price label is missing, and will evaluate comments.
          */
         requirePriceLabel: T.Boolean({ default: true }),
+        limitRewards: T.Boolean({
+          default: true,
+          description: "Should the rewards of non-assignees be limited to the issue reward?",
```

Comment:
```suggestion
          description: "Should the rewards of non-assignees be limited to the task reward?",
```

Isn't this redundant?